nbbuild/rat-exclusions.txt

```diff
@@ -146,6 +150,104 @@ ide/properties/src/org/netbeans/modules/properties/sample.properties.template
 java/selenium2.java/src/org/netbeans/modules/selenium2/java/SeleneseIT.java.template
 java/testng.ui/src/org/netbeans/modules/testng/ui/resources/*.template
 websvccommon/websvc.saas.api/src/org/netbeans/modules/websvc/saas/oauth/*.template
+# cnd template files
```

Comment:
Is it really not possible to include the ALv2 license in each of these files? We had other cases where the templates are processed and the template language allowed to have the licenses header and the licenses would not show up in the generated file (freemarker for example allows to place comments in files).

In anycase, I'd like to prefer to use the licenseinfo.xml facitily to document, that the files are actually ALv2 and not just plain ignored (the build system reads licenseinfo.xml and generates a report from that).